example/extensions/lib_subgraph/subgraph_lib.cc

```diff
@@ -0,0 +1,250 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2019 by Contributors
+ * \file subgraph_lib.cc
+ * \brief subgraph operator implementation library file
+ */
+
+#include <math.h>
+#include <iostream>
+#include <algorithm>
+#include "lib_api.h"
+
+/* function to execute log operator on floats */
+void myLog(MXTensor &in, MXTensor &out) {
```

Comment:
I think here we were going with lowerUpper camel casing for global functions, and LowerUpper for class names and member functions.

Remember this is an example of a user's custom library. So not code that will be compiled into libmxnet.so.

Reply:
I didn't know we have a convention like that. Shall we follow google's cpp coding style guide to have CamelCase for all functions? It's better to have a consistent cpp coding style in the repo.

Reply:
I agree we should have a convention in the repo. But this is not the PR to decide that. There are already multiple conventions in use in MXNet. Once a convention is agreed upon in dev-mailing-list we should stick to it in PR reviews. But since we dont have one yet, theres no reason to require it for some PRs (like this one) and not others. Lets do this right and  be consistent. 

Will you propose a style/standard on dev-mailing-list so we can follow up?